ow_plexil/src/plans/InitializeAntennaAndArm.plp

```diff
@@ -0,0 +1,27 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Reset pan and tilt angles to 0 and stow the arm (if not already there)
+
+#include "plan-interface.h"
+
+InitializeAntennaAndArm:
+{
+  InOut Boolean initialize;
```

Comment:
I really don't think this variable should be here, as it complicates this plan and the calling plans.   You mentioned concurrence support as the reason, but a cleaner way to that is factor the calls to this node out of the Concurrence.

Reply:
Will that work when the concurrence node is the root node? Currently, all the patterns/plans more complex than Pattern 1 have a concurrence node as the root.